rasa/nlu/featurizers/dense_featurizer/lm_featurizer.py

```diff
@@ -42,58 +45,102 @@
-            skip_model_load: Skip loading the model for pytests.
-        """
-        super(LanguageModelFeaturizer, self).__init__(component_config)
+        """Initializes LanguageModelFeaturizer with the model
+        specified in the config."""
+        super(LanguageModelFeaturizerGraphComponent, self).__init__(
+            execution_context.node_name, config
+        )
+        self._config = {**self.get_default_config(), **config}
+        self._model_storage = model_storage
+        self._resource = resource
+        self._execution_context = execution_context
+        self._check_language_is_supported()
         self._load_model_metadata()
-        self._load_model_instance(skip_model_load)
+        self._load_model_instance(self._config["skip_model_load"])
+
+    @staticmethod
+    def get_default_config() -> Dict[Text, Any]:
+        """Returns LanguageModelFeaturizer's default config."""
+        return {
+            **DenseFeaturizer2.get_default_config(),
+            # name of the language model to load.
+            "model_name": "bert",
+            # Pre-Trained weights to be loaded(string)
+            "model_weights": None,
+            # an optional path to a specific directory to download
+            # and cache the pre-trained model weights.
+            "cache_dir": None,
+            "skip_model_load": False,
+            "language": None,
+        }
+
+    @classmethod
+    def validate_config(cls, config: Dict[Text, Any]) -> None:
+        """Validates that the component is configured properly."""
+        pass
+
+    @classmethod
+    def validate_compatibility_with_tokenizer(
+        cls, config: Dict[Text, Any], tokenizer_type: Type[Tokenizer]
+    ) -> None:
+        """Validates that the featurizer is compatible with the given tokenizer."""
+        pass
 
     @classmethod
     def create(
-        cls, component_config: Dict[Text, Any], config: RasaNLUModelConfig
-    ) -> "DenseFeaturizer":
-        language = config.language
-        if not cls.can_handle_language(language):
-            # check failed
-            raise UnsupportedLanguageError(cls.name, language)
-        return cls(component_config)
+        cls,
+        config: Dict[Text, Any],
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
+    ) -> "LanguageModelFeaturizerGraphComponent":
+        """Creates a LanguageModelFeaturizer by loading the model
+        specified in the config."""
+        return cls(config, model_storage, resource, execution_context)
 
-    def _load_model_metadata(self) -> None:
-        """Load the metadata for the specified model and sets these properties.
+    @staticmethod
+    def supported_languages() -> Optional[List[Text]]:
+        """Returns list of supported languages."""
+        # return ["en"]
+        # None means all languages are supported
+        return None  # TODO: check which language are supported
```

Comment:
I couldn't figure out what languages were supported in the original implementation 🤔 I left it as `None` to mean all languages are supported. If this is correct, I'll remove `# return ["en"]` and the `# TODO`. 
Maybe the support comes from Hugging face and not us?

Reply:
Seems there was no check before so you can just omit the `supported_languages` implementation in favor of the one in the parent class